packages/common/src/cursorlessCommandIds.ts

```diff
@@ -0,0 +1,107 @@
+export const CURSORLESS_COMMAND_ID =
+  "cursorless.command" satisfies CursorlessCommandId;
+
+export interface CommandDescription {
+  readonly isVisible: boolean;
+  readonly title: string;
+}
+
+abstract class Command {
+  constructor(private baseTitle: string) {}
+
+  get title(): string {
+    return `Cursorless: ${this.baseTitle}`;
+  }
+}
+
+class HiddenCommand extends Command implements CommandDescription {
+  readonly isVisible = false as const;
+}
+
+class VisibleCommand extends Command implements CommandDescription {
+  readonly isVisible = true as const;
+}
```

Comment:
Could probably simplify these to a single class, but was planning to add argument schema, and require that visible commands can be run with no args.  Maybe I should wait until I do that before introducing this complexity?  Idk it's only a few lines